package.json

```diff
@@ -21,26 +21,15 @@
     "prod": "rm -rf dist && NODE_ENV=production webpack -p --progress --display-modules --config webpack.prod.config"
   },
   "dependencies": {
-    "@nypl/dgx-svg-icons": "0.3.4",
     "bootstrap": "^3.3.6",
-    "font-awesome": "^4.6.3",
-    "isomorphic-fetch": "^2.2.1",
-    "numeral": "^2.0.6",
-    "opds-feed-parser": "0.0.14",
     "opds-web-client": "0.1.27",
```

Comment:
I'm not sure if it's worth having this as a dependency but I haven't looked at what all it's used for yet.

Reply:
The DataFetcher, RequestError, and BaseActionCreator are being imported from it.

Reply:
It kind of seems like overkill to depend on the whole app just for those. If they're that useful maybe they should eventually become a separate package.

Reply:
I think they should definitely become a separate package!  But that's a conversation for another branch; let's leave it as is for right now.  (There's no risk of it hurting anything in the meantime...right?)

Reply:
Yeah that's fine.

Reply:
Don't forget to remove later!